cardano-node/src/Cardano/Node/Configuration/CLI.hs

```diff
@@ -1,433 +0,0 @@
-           ( long "noncritical-cq"
-          <> metavar "QUALITY"
-          <> help "If chain quality after bootstrap era is less than this value, non critical misbehavior will be reported."
-           )
-        <*> lastDoubleOption
-           ( long "critical-cq"
-          <> metavar "QUALITY"
-          <> help "If chain quality after bootstrap era is less than this value, critical misbehavior will be reported."
-           )
-        <*> lastIntOption
-           ( long "critical-fork-threshold"
-          <> metavar "BLOCKS"
-          <> help "Number of blocks such that if so many blocks are rolled back, it requires immediate reaction."
-           )
-        <*> lastIntOption
-           ( long "fixed-time-cq"
-          <> metavar "TIME"
-          <> help "Chain quality will be also calculated for this amount of seconds."
-           )
-
---------------------------------------------------------------------------------
--- Certificates
---------------------------------------------------------------------------------
-
--- | TLS CLI Parser.
-configTLSCLIParser :: Parser PartialTLS
-configTLSCLIParser =
-    PartialTLS
-        <$> configCertificateCLIParser
-        <*> configCertificateCLIParser
-        <*> configCertificateCLIParser
-
--- | Certificate CLI parser.
-configCertificateCLIParser :: Parser PartialCertificate
-configCertificateCLIParser =
-    PartialCertificate
-        <$> lastStrOption
-           ( long "cert-organization-name"
-          <> metavar "CERT-ORGANIZATION-NAME"
-          <> help "Certificate organization."
-           )
-        <*> lastStrOption
-           ( long "cert-common-name"
-          <> metavar "CERT-COMMON-NAME"
-          <> help "Certificate common name."
-           )
-        <*> lastIntOption
-           ( long "cert-expiry-days"
-          <> metavar "CERT-EXPIRY-DAYS"
-          <> help "Certificate days of expiration."
-           )
-        <*> lastTextListOption
-           ( long "cert-alternative-dns"
-          <> metavar "CERT-ALTERNATIVE-DNS"
-          <> help "Certificate alternative DNS."
-           )
-
---------------------------------------------------------------------------------
--- Wallet
---------------------------------------------------------------------------------
-
--- | Wallet CLI parser.
-configWalletCLIParser :: Parser PartialWallet
-configWalletCLIParser =
-    PartialWallet
-        <$> lastBoolOption
-           ( long "th-enabled"
-          <> metavar "TH-ENABLED"
-          <> help "Throttle enabled/disabled."
-           )
-        <*> lastIntOption
-           ( long "th-rate"
-          <> metavar "TH-RATE"
-          <> help "Throttle rate."
-           )
-        <*> lastStrOption
-           ( long "th-period"
-          <> metavar "TH-PERIOD"
-          <> help "Throttle period."
-           )
-        <*> lastIntOption
-           ( long "th-burst"
-          <> metavar "TH-BURST"
-          <> help "Throttle burst."
-           )
```

Comment:
Where did the contents of this file go? I'm not seeing them relocated anywhere.

Reply:
The exported functions weren't being used anywhere. I need @deepfire to confirm if this is dead code.

Reply:
Ah ok, nice!